### DISCUSSION TITLE
SI5351 frequency setting

### DISCUSSION THREAD

**Discussion by holdenweb** [micropython/micropython#15689] [Hardware & Peripherals]
I have a fairly well-calibrated oscilloscope

The 'scope tells me that when I set the frequency to 455kHz I actually get a signal at 455.139 kHz.

To obtain what the 'scope tells me is a 455kHz signal I have to set the frequency at 454.861 kHz.

Is there some standard calibration procedure I can apply, or do I need to recalibrate my'scope?

**Comment by davefes** [comment]
What Oscilloscope are you using?   Has it got a built-in frequency counter?  Displaying 0.001KHz suggests it has.  If the oscilloscope has an any sort of frequency reference I'd be surprised that it was that much off-frequency.
